### PR TITLE
Lj 53 report a problem

### PR DESCRIPTION
**Pull request by krischarbonneau**
# Description

[LJ-53 Report a problem component (non-functional)](https://lj-decd.atlassian.net/browse/LJ-53?atlOrigin=eyJpIjoiZDUzMDE2YWEwMDdkNDg3N2JlNTM4NGIwZjI1MzNlYzkiLCJwIjoiaiJ9 )

Opening a draft PR so conversations can begin on this task. Most recent commit has brought in the ActionButton and CheckBox (not completed, so not implemented currently) from Alpha Site and a DropDown component.

## Test Instructions

1. Pull in branch
1. Type ```npm run dev```

## Meets Definition of Done

- [ ] Meets design spec
- [ ] unit tests are included

### CODE REVIEW

File: components/atoms/ReportProblem.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <div className="pb-6 pt-6">
+      <DropDown text="Report a problem or mistake on this page">
+        <div className="pt-2">
+          <form action="" method="post">
+            <div className="container border border-gray-md rounded bg-gray-light-report pb-5">
+              <fieldset className="text-left pl-2">
+                <legend className="text-center pb-3">
+                  <span>
+                    <h3>
+                      <hr className="border-1 pb-3 border-gray-300" />
```
Comment (pcraig3) [inline components/atoms/ReportProblem.js:17]: Why the rule? Seems like we can drop this.

**Review by pcraig3 — COMMENTED** [review]
Overall this looks pretty good, but there's some weird button stuff going on here.

1. We are importing the `ActionButton`, but doesn't look like we're using it.
2. We are creating a `Button` component
3. We have a `<button>` wrapping the `<summary>` element. 

- What's the difference between ActionButton and Button?
- Can we remove the button surrounding the details element?

File: components/atoms/ReportProblem.js (hunk 1 above)
Comment (krischarbonneau) [reply components/atoms/ReportProblem.js:17]: Was basing it off of Canada.ca's design as they have a rule at the top. Can take it out if not necessary though
Screenshot 2021-04-21 120749

**Comment by krischarbonneau** [comment]
> Overall this looks pretty good, but there's some weird button stuff going on here.
> 
> 1. We are importing the `ActionButton`, but doesn't look like we're using it.
> 2. We are creating a `Button` component
> 3. We have a `<button>` wrapping the `<summary>` element.
> 
> * What's the difference between ActionButton and Button?
> * Can we remove the button surrounding the details element?

Originally I was using ActionButton for the submit button, however I was having issues overriding the styles on the ActionButton so it was way too big. I can revert back to the ActionButton if this can be resolved though.

I can get rid of the button wrapping the details element, yep.

File: components/atoms/ReportProblem.js (hunk 1 above)
Comment (pcraig3) [reply components/atoms/ReportProblem.js:17]: Yeah, I see it. It's not on the design system version, so I don't think we need it: https://design.canada.ca/common-design-patterns/report-problem.html 

Weird that they are using it on Canada.ca though.

**Comment by pcraig3** [comment]
Yeah, let's get rid of the wrapping button and delete the `ActionButton`.

**Comment by pcraig3** [comment]
I think this looks pretty good as you've implemented it. I opened an issue in the other repo about some improvements to be made there, so we shouldn't copy that one as-is: https://github.com/DTS-STN/Alpha-Site/issues/42

**Comment by pcraig3** [comment]
The two things I've noticed are that there is no `:active` or `:hover` states for the summary element, which the Canada design system has.

**Review by pcraig3 — APPROVED** [review]
This is super good work! Thanks @krischarbonneau!

**Pull request merged**